src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
```

Comment:
it is better to use records for DTO

if your Checkstyle fires at it - upgrade `maven-checkstyle-plugin` to 3.3.0